operation/fhir-operation-healthcheck/src/main/java/com/ibm/fhir/operation/healthcheck/HealthcheckOperation.java

```diff
@@ -50,11 +52,16 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
 
             FHIRPersistenceTransaction tx = resourceHelper.getTransaction();
             tx.begin();
-            
+
             try {
                 OperationOutcome operationOutcome = pl.getHealth();
                 checkOperationOutcome(operationOutcome);
-                return FHIROperationUtil.getOutputParameters(operationOutcome);
+
+                if (FHIRRequestContext.get().getReturnPreference() == HTTPReturnPreference.OPERATION_OUTCOME) {
```

Comment:
should this be a left handed equals?  then it's resilient to a null/ or is Operation_Outcome an enum?